chkcryptoboot_install

```diff
@@ -30,6 +30,16 @@ build ()
 	      cd $TMPDIR
 	      sha512sum disk-head > $TMPDIR/disk-head.sha512
 	      add_file $TMPDIR/disk-head.sha512 /etc/chkcryptoboot
+
+	      if [ -n "$BOOT_PARTITION" -a -b ${BOOT_PARTITION} ]; then
+		  #read first 62 sectors from BIOS boot partition
+		  dd if=$BOOT_PARTITION of=$TMPDIR/grub-core bs=512 count=62
```

Comment:
While I'm ok with hashing the entire BOOT_PARTITION, grub's core.img uses precisely 25,389 bytes, and this is hardcoded on their code, if a img turns out bigger than that, it fails. We can consider this for improvement on a future release.